app/lib/services/voip/MediaCallEvents.test.ts

```
@@ -58,10 +138,20 @@ jest.mock('react-native-callkeep', () => ({
-		applyRestStateSignals: jest.fn(() => Promise.resolve())
+		applyRestStateSignals: jest.fn(() => Promise.resolve()),
+		registerOnInitComplete: jest.fn()
 	}
 }));
 
+jest.mock('./clearPendingCallView', () => ({
+	clearPendingCallView: jest.fn()
+}));
+
+jest.mock('../../navigation/appNavigation', () => ({
+    __esModule: true,
```

ESLint

[error] Replace `····` with `↹`

(prettier/prettier)